client/stats/sender.go

```diff
@@ -30,6 +29,16 @@ import (
 
 const statsSenderLogPrefix = "[session-stats-sender] "
 
+// State represents the state changes in connection that the sender is interested in
+type State int
+
+const (
+	// Connected represents the connected state
+	Connected State = iota
```

Comment:
What is wrong with constants defined in connection manager?
Stats sender is aware and handles same connection states